changelogs/unreleased/config-log-modules-removal.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/config
+
+* Fix handling of `log.modules` option removal (gh-10728).
```

Comment:
```suggestion
* Fixed handling of `log.modules` option removal (gh-10728).
```

Reply:
Thanks, really. Forgot about that. Fixed.